packages/angular/pwa/BUILD.bazel

```diff
@@ -7,6 +7,11 @@ load("@npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
 load("//tools:defaults.bzl", "ts_library")
 load("//tools:ts_json_schema.bzl", "ts_json_schema")
 
+# @external_begin
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
+# @external_end
```

Comment:
I added these external to follow the existing convention, but I'm not sure what they are used for.

Reply:
I was going to ask. It doesn't seem like there is any value in having these as the CLI build files are not exposed publicly anyway.

Reply:
These comments were added by @kyliau and were needed for a G3 sync.

Reply:
interesting. we should probably re-evaluate (or if @josephperrott already did), whether this is still something we want to maintain long-term. I suspect it will complicate the Bazel migration/maintaince.

Reply:
This is something we do not need to include as we do not use our BUILD files internally in g3, we have google3 specific ones.

Reply:
Drive by comment: I'm hoping to reevaluate how we integrate the CLI internally to improve the syncing story so we can focus just on the external side here.